README.md

```diff
@@ -1 +1,179 @@
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [ Walkthrougth](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Tropical_Gateways] <a name="about-project"></a>
+
+**[Tropical_Gateways]** is a web application for the first capstone project of the first module, it emulates a travel agency based on a template web we were given.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+In this project we used HTML CSS and Javascript
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Mobile_responsive]**
+- **[Flexbox]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- You can watch the [video](https://www.loom.com/share/5beaaf62640445b3baf91d8969024eac?sid=f7e8bbc7-e808-4f29-924b-d3095c4e467c) walkthrough here.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+
+```sh
+ npm install -y
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+```sh
+  cd tropical_gateways
+  git clone git@github.com:Rebelzob/Tropical_Gateways.git
+```
+
+### Install
+
+Install this project with:
+
+
+
+```sh
+  cd tropical_gateways
+  npm init -y
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+
+```sh
+  npm run
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@Rebelzob](https://github.com/Rebelzob)
+- LinkedIn: [Karim-Barragan](https://www.linkedin.com/in/karim-barragan/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ x] **[Desktop_version]**
+- [ x] **[JS_implementation]**
+- [ ] **[Back_end_implementation]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project follow and give me a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
- [ ] Please kindly add Appropriate credit to Cindy Shin in your Acknowledgments section, as required by the Creative Commons License. For Example:

## EXAMPLE
`This template was inspired by [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
`